doc/installing.md

```diff
@@ -66,6 +66,49 @@ If you prefer a different installation method, see {ref}`installing`.
    This will create a minimal setup with default options.
    If you want to tune the initialization options, see {ref}`initialize` for more information.
 
+#### Gentoo
+Incus and all of its dependencies are available in Gentoo's main repository as [`app-containers/incus`](https://packages.gentoo.org/packages/app-containers/incus).
+
+Install Incus with:
+
+    emerge -av app-containers/incus
+
+Note: Installing LTS vs. feature-release will be explained later, when Incus upstream and Gentoo's repository has those releases available.
+
+There will be two newly created groups associated to Incus: 
+`incus` to allow basic user access (launch containers), and `incus-admin` for `incus admin` controls. Add your regular users to either, or both, depending on your setup and use cases.
+
+After installation, you may want to configure Incus. This is optional though, as the defaults should also just work.
+
+ - **openrc**: edit /etc/conf.d/incus
+
+ - **systemd**: systemctl edit --full lxd.service
```

Comment:
@juippis that probably should be `incus.service`?

Reply:
Oops, had to spawn a systemd vm which had lxd installed to check correct **systemctl** syntax. Sharp eyes!